SOURCES/0004-feat-drivers-add-CephFS-GlusterFS-and-XFS-drivers.patch

```diff
@@ -702,10 +702,6 @@ index 0000000..15dbe44
 +            raise xs_errors.XenError('ConfigDeviceMissing')
 +
 +        self.root = self.dconf['device']
-+        for dev in self.root.split(','):
-+            if not self._isvalidpathstring(dev):
-+                raise xs_errors.XenError('ConfigDeviceInvalid', \
-+                      opterr='path is %s' % dev)
```

Comment:
I suppose this removal reflects similar changes in the upstream drivers?

Also, were there other changes in the upstream drivers that would be worth applying to ours?

Reply:
Answer: yes, it was an upstream change in other drivers, and no, Ronan hasn't seen other changes having an impact on our drivers.